### DISCUSSION TITLE
[Feature request] Horizontal split view and stack split windows

### DISCUSSION THREAD

**Discussion by Terkyz** [Floorp-Projects/Floorp#749] [Feature Request]
Actually Floorp possesses a great feature that I thought would never come to any Gecko-Based browser at that is the vertical split view. I used to love this feature from Vivaldi but I didn't like Vivaldi at all.

Vivaldi actually have horizontal and vertical split views and it's super useful. My request is the implementation of the horizontal layout, and it would be great if you would be able to stack them.

This feature is awesome, although I think that it may need a little bit of polish. Overall, it's a great feature!

I swear, I search for similar discussions, but I cannot find anything related.

**Comment by akeeton** [comment]
This feature is available, look for the split-view icon in the address bar and choose "Vertical" (not "horizontal" as you described it, but the effect is the same).  I just installed Floorp, so this may be a new feature since you posted this discussion.
![image](https://github.com/user-attachments/assets/32ec0081-144a-46e3-b9a9-692a54aea9fc)

**Reply by Terkyz** [reply]
I didn't know, thank you for telling me.
